pkg/dao/repository_configs.go

```diff
@@ -375,6 +375,22 @@ func (r repositoryConfigDaoImpl) Update(orgID, uuid string, repoParams api.Repos
 	return updatedUrl, nil
 }
 
+// UpdateLastSnapshotTask updates the RepositoryConfig with the latest SnapshotTask
+func (r repositoryConfigDaoImpl) UpdateLastSnapshotTask(taskUUID string, orgID string, repoUUID string) error {
+	result := r.db.Exec(`
+			UPDATE repository_configurations 
+			SET last_snapshot_task_uuid = ? 
+			WHERE repository_configurations.repository_uuid = ?`,
+		taskUUID,
+		repoUUID,
+	)
```

Comment:
you want to also filter by orgID here, right? It looks like you included it in the method params, but forgot to use it.

Reply:
⚠️  Good call out!! ugh.. so used to typescript screaming for unused variables.